tests/internal.spec.js

```diff
@@ -22,19 +22,19 @@ describe('Can handle', function () {
   it('missing username', function () {
     expect(function () {
       nclWrapper();
-    }).to.throw('AssertionError');
+    }).to.throw();
```

Comment:
should we use the error message here:`
.to.throw('must include username in auth')`